### PR TITLE
added serve functionality for gulpfile

### PR DESCRIPTION
**Pull request by chenshay**
#3

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -29,6 +29,30 @@ gulp.task('build', function() {
       .pipe(gulp.dest('dist'));
 });
 
+gulp.task('clean', function() {
+  return del(['dist']);
+});
+
+function serve() {
+  var app = require('express')();
+  var webserver = require('gulp-webserver');
```
Comment (erwinmombay) [inline gulpfile.js]: the dependency doesnt exist.

run `yarn add --dev gulp-webserver` and `yarn add -dev express`

Comment (chenshay) [inline gulpfile.js]: done

File: package.json [hunk 2]
```diff
@@ -2,6 +2,10 @@
   "devDependencies": {
     "babel-preset-env": "^1.5.1",
     "gulp": "^3.9.1",
-    "gulp-babel": "^6.1.2"
+    "gulp-babel": "^6.1.2",
+    "gulp-webserver": "^0.9.1"
```
Comment (erwinmombay) [inline package.json]: you missed adding `del`

Comment (chenshay) [inline package.json]: done

**Pull request merged**